src/main/resources/assets/api-docs.yaml

```diff
@@ -122,6 +122,8 @@ paths:
       responses:
         404:
           description: Deprecated endpoint. Use  `GET /api/tdr/{identifier}` instead.
+  /api/daa/{daaId}:
+    $ref: 'paths/daaByDaaId.yaml'
   /api/daa/{dacId}:
     $ref: 'paths/daaById.yaml'
```

Comment:
We can't actually do this ... we might need a refactor of `/api/daa/{dacId}:` to be something like `/api/daa/dac/{dacId}:` or something similar. The two paths (as defined) have the same signature (`/api/daa/<integer>`) so Jersey cannot distinguish between the two.

Reply:
For now, I've refactored the POST api signature.